[ValidationError]: OBPv4.0.0-deleteBankLevelDynamicMessageDoc

{
   "type": "dict_type",
   "loc": [
      "success_response_body"
   ],
   "msg": "Input should be a valid dictionary",
   "input": true,
   "url": "https://errors.pydantic.dev/2.10/v/dict_type"
}
------------------------------------------------------------------------------------------------------------------------


[ValidationError]: OBPv4.0.0-deleteMyApiCollection

{
   "type": "dict_type",
   "loc": [
      "success_response_body"
   ],
   "msg": "Input should be a valid dictionary",
   "input": true,
   "url": "https://errors.pydantic.dev/2.10/v/dict_type"
}
------------------------------------------------------------------------------------------------------------------------


[ValidationError]: OBPv4.0.0-deleteProductFee

{
   "type": "dict_type",
   "loc": [
      "success_response_body"
   ],
   "msg": "Input should be a valid dictionary",
   "input": true,
   "url": "https://errors.pydantic.dev/2.10/v/dict_type"
}
------------------------------------------------------------------------------------------------------------------------


[ValidationError]: OBPv4.0.0-updateAuthenticationTypeValidation

{
   "type": "dict_type",
   "loc": [
      "example_request_body"
   ],
   "msg": "Input should be a valid dictionary",
   "input": [
      "DirectLogin",
      "OAuth1.0a",
      "GatewayLogin",
      "DAuth",
      "OAuth2_OIDC",
      "OAuth2_OIDC_FAPI"
   ],
   "url": "https://errors.pydantic.dev/2.10/v/dict_type"
}
------------------------------------------------------------------------------------------------------------------------


[ValidationError]: OBPv4.0.0-deleteDynamicResourceDoc

{
   "type": "dict_type",
   "loc": [
      "success_response_body"
   ],
   "msg": "Input should be a valid dictionary",
   "input": true,
   "url": "https://errors.pydantic.dev/2.10/v/dict_type"
}
------------------------------------------------------------------------------------------------------------------------


[ValidationError]: OBPv4.0.0-deleteAuthenticationTypeValidation

{
   "type": "dict_type",
   "loc": [
      "success_response_body"
   ],
   "msg": "Input should be a valid dictionary",
   "input": true,
   "url": "https://errors.pydantic.dev/2.10/v/dict_type"
}
------------------------------------------------------------------------------------------------------------------------


[ValidationError]: OBPv4.0.0-deleteEndpointMapping

{
   "type": "dict_type",
   "loc": [
      "success_response_body"
   ],
   "msg": "Input should be a valid dictionary",
   "input": true,
   "url": "https://errors.pydantic.dev/2.10/v/dict_type"
}
------------------------------------------------------------------------------------------------------------------------


[ValidationError]: OBPv4.0.0-deleteMyApiCollectionEndpointById

{
   "type": "dict_type",
   "loc": [
      "success_response_body"
   ],
   "msg": "Input should be a valid dictionary",
   "input": true,
   "url": "https://errors.pydantic.dev/2.10/v/dict_type"
}
------------------------------------------------------------------------------------------------------------------------


[ValidationError]: OBPv4.0.0-deleteTransactionRequestAttributeDefinition

{
   "type": "dict_type",
   "loc": [
      "success_response_body"
   ],
   "msg": "Input should be a valid dictionary",
   "input": true,
   "url": "https://errors.pydantic.dev/2.10/v/dict_type"
}
------------------------------------------------------------------------------------------------------------------------


[ValidationError]: OBPv4.0.0-getSystemLevelEndpointTags

{
   "type": "dict_type",
   "loc": [
      "success_response_body"
   ],
   "msg": "Input should be a valid dictionary",
   "input": [
      {
         "bank_id": "gh.29.uk",
         "endpoint_tag_id": "7uy8a7e4-6d02-40e3-a129-0b2bf89de8uh",
         "operation_id": "OBPv4.0.0-getBanks",
         "tag_name": "BankAccountTag1"
      }
   ],
   "url": "https://errors.pydantic.dev/2.10/v/dict_type"
}
------------------------------------------------------------------------------------------------------------------------


[ValidationError]: OBPv4.0.0-createAuthenticationTypeValidation

{
   "type": "dict_type",
   "loc": [
      "example_request_body"
   ],
   "msg": "Input should be a valid dictionary",
   "input": [
      "DirectLogin",
      "OAuth1.0a",
      "GatewayLogin",
      "DAuth",
      "OAuth2_OIDC",
      "OAuth2_OIDC_FAPI"
   ],
   "url": "https://errors.pydantic.dev/2.10/v/dict_type"
}
------------------------------------------------------------------------------------------------------------------------


[ValidationError]: OBPv4.0.0-deleteBankLevelDynamicResourceDoc

{
   "type": "dict_type",
   "loc": [
      "success_response_body"
   ],
   "msg": "Input should be a valid dictionary",
   "input": true,
   "url": "https://errors.pydantic.dev/2.10/v/dict_type"
}
------------------------------------------------------------------------------------------------------------------------


[ValidationError]: OBPv4.0.0-deleteSystemLevelEndpointTag

{
   "type": "dict_type",
   "loc": [
      "success_response_body"
   ],
   "msg": "Input should be a valid dictionary",
   "input": true,
   "url": "https://errors.pydantic.dev/2.10/v/dict_type"
}
------------------------------------------------------------------------------------------------------------------------


[ValidationError]: OBPv4.0.0-getBankLevelEndpointTags

{
   "type": "dict_type",
   "loc": [
      "success_response_body"
   ],
   "msg": "Input should be a valid dictionary",
   "input": [
      {
         "bank_id": "gh.29.uk",
         "endpoint_tag_id": "7uy8a7e4-6d02-40e3-a129-0b2bf89de8uh",
         "operation_id": "OBPv4.0.0-getBanks",
         "tag_name": "BankAccountTag1"
      }
   ],
   "url": "https://errors.pydantic.dev/2.10/v/dict_type"
}
------------------------------------------------------------------------------------------------------------------------


[ValidationError]: OBPv4.0.0-deleteDynamicMessageDoc

{
   "type": "dict_type",
   "loc": [
      "success_response_body"
   ],
   "msg": "Input should be a valid dictionary",
   "input": true,
   "url": "https://errors.pydantic.dev/2.10/v/dict_type"
}
------------------------------------------------------------------------------------------------------------------------


[ValidationError]: OBPv5.1.0-createUserWithAccountAccessById

{
   "type": "dict_type",
   "loc": [
      "success_response_body"
   ],
   "msg": "Input should be a valid dictionary",
   "input": [
      {
         "id": "1234",
         "short_name": "short_name",
         "description": "description",
         "metadata_view": "owner",
         "is_public": true,
         "is_system": true,
         "alias": "No",
         "hide_metadata_if_alias_used": true,
         "can_add_comment": true,
         "can_add_corporate_location": true,
         "can_add_image": true,
         "can_add_image_url": true,
         "can_add_more_info": true,
         "can_add_open_corporates_url": true,
         "can_add_physical_location": true,
         "can_add_private_alias": true,
         "can_add_public_alias": true,
         "can_add_tag": true,
         "can_add_url": true,
         "can_add_where_tag": true,
         "can_delete_comment": true,
         "can_add_counterparty": true,
         "can_delete_corporate_location": true,
         "can_delete_image": true,
         "can_delete_physical_location": true,
         "can_delete_tag": true,
         "can_delete_where_tag": true,
         "can_edit_owner_comment": true,
         "can_see_bank_account_balance": true,
         "can_query_available_funds": true,
         "can_see_bank_account_bank_name": true,
         "can_see_bank_account_currency": true,
         "can_see_bank_account_iban": true,
         "can_see_bank_account_label": true,
         "can_see_bank_account_national_identifier": true,
         "can_see_bank_account_number": true,
         "can_see_bank_account_owners": true,
         "can_see_bank_account_swift_bic": true,
         "can_see_bank_account_type": true,
         "can_see_comments": true,
         "can_see_corporate_location": true,
         "can_see_image_url": true,
         "can_see_images": true,
         "can_see_more_info": true,
         "can_see_open_corporates_url": true,
         "can_see_other_account_bank_name": true,
         "can_see_other_account_iban": true,
         "can_see_other_account_kind": true,
         "can_see_other_account_metadata": true,
         "can_see_other_account_national_identifier": true,
         "can_see_other_account_number": true,
         "can_see_other_account_swift_bic": true,
         "can_see_owner_comment": true,
         "can_see_physical_location": true,
         "can_see_private_alias": true,
         "can_see_public_alias": true,
         "can_see_tags": true,
         "can_see_transaction_amount": true,
         "can_see_transaction_balance": true,
         "can_see_transaction_currency": true,
         "can_see_transaction_description": true,
         "can_see_transaction_finish_date": true,
         "can_see_transaction_metadata": true,
         "can_see_transaction_other_bank_account": true,
         "can_see_transaction_start_date": true,
         "can_see_transaction_this_bank_account": true,
         "can_see_transaction_type": true,
         "can_see_url": true,
         "can_see_where_tag": true,
         "can_see_bank_routing_scheme": true,
         "can_see_bank_routing_address": true,
         "can_see_bank_account_routing_scheme": true,
         "can_see_bank_account_routing_address": true,
         "can_see_other_bank_routing_scheme": true,
         "can_see_other_bank_routing_address": true,
         "can_see_other_account_routing_scheme": true,
         "can_see_other_account_routing_address": true,
         "can_add_transaction_request_to_own_account": true,
         "can_add_transaction_request_to_any_account": true,
         "can_see_bank_account_credit_limit": true,
         "can_create_direct_debit": true,
         "can_create_standing_order": true
      }
   ],
   "url": "https://errors.pydantic.dev/2.10/v/dict_type"
}
------------------------------------------------------------------------------------------------------------------------


[ValidationError]: OBPv4.0.0-deleteMyApiCollectionEndpoint

{
   "type": "dict_type",
   "loc": [
      "success_response_body"
   ],
   "msg": "Input should be a valid dictionary",
   "input": true,
   "url": "https://errors.pydantic.dev/2.10/v/dict_type"
}
------------------------------------------------------------------------------------------------------------------------


[ValidationError]: OBPv4.0.0-deleteMyApiCollectionEndpointByOperationId

{
   "type": "dict_type",
   "loc": [
      "success_response_body"
   ],
   "msg": "Input should be a valid dictionary",
   "input": true,
   "url": "https://errors.pydantic.dev/2.10/v/dict_type"
}
------------------------------------------------------------------------------------------------------------------------


[ValidationError]: OBPv4.0.0-deleteBankLevelEndpointMapping

{
   "type": "dict_type",
   "loc": [
      "success_response_body"
   ],
   "msg": "Input should be a valid dictionary",
   "input": true,
   "url": "https://errors.pydantic.dev/2.10/v/dict_type"
}
------------------------------------------------------------------------------------------------------------------------


[ValidationError]: OBPv4.0.0-deleteJsonSchemaValidation

{
   "type": "dict_type",
   "loc": [
      "success_response_body"
   ],
   "msg": "Input should be a valid dictionary",
   "input": true,
   "url": "https://errors.pydantic.dev/2.10/v/dict_type"
}
------------------------------------------------------------------------------------------------------------------------


[ValidationError]: OBPv4.0.0-deleteBankLevelEndpointTag

{
   "type": "dict_type",
   "loc": [
      "success_response_body"
   ],
   "msg": "Input should be a valid dictionary",
   "input": true,
   "url": "https://errors.pydantic.dev/2.10/v/dict_type"
}
------------------------------------------------------------------------------------------------------------------------


